testsuite/classlibrary/TestFunction.sc

```diff
@@ -66,6 +72,8 @@ CommonTestClass {
 		}
 	}
 
+	*reset { x = nil; y = 7; }
+	reset { a = nil; b = 3; }
```

Comment:
yes, depending on when you call them, they either tear down after a test, or setup before the test, so I thought reset is the most general term for both: reset the instance or the class to its initial state.